google/cloud/internal/streaming_write_rpc_logging.h

```diff
@@ -67,7 +67,8 @@ class StreamingWriteRpcLogging
       GCP_LOG(DEBUG) << prefix << "() << "
                      << DebugString(*result, tracing_options_);
     } else {
-      GCP_LOG(DEBUG) << prefix << "() << " << result.status();
+      GCP_LOG(DEBUG) << prefix << "() << "
+                     << DebugString(result.status(), tracing_options_);
```

Comment:
Indeed, that is next on my list.

Reply:
On second thoughts, I don't think this is worth it, if only because of the way we currently add `>> response=` and `>> status=` prefixes in some cases.  It doesn't seem right to move that into a `DebugString(StatusOr<T>, options)` without a broader convention about how `DebugString()` should be used, and I'll leave that for another day.